app/src/activity.c

```diff
@@ -34,7 +34,8 @@ int raise_event() {
         (struct zmk_activity_state_changed){.state = activity_state}));
 }
 
-int set_state(enum zmk_activity_state state) {
+int activity_set_state(enum zmk_activity_state state) {
```

Comment:
I think you have to rename all of these `activity_set_state`s to `zmk_activity_set_state`.